### PR TITLE
Create signer everytime after fetching app config

### PR DESCRIPTION
**Pull request by shrinathprabhu**
## Changes

- Signer was not getting updated after creating new app. So added `createTransactionSigner`, everytime we do fetch app details

## Checklist

- [x] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

### CODE REVIEW

File: src/pages/AppConfigure.vue [hunk 1]
```diff
@@ -107,6 +108,7 @@ function handleSmartContractErrors(type: string, error: unknown) {
 
 function handleCancel() {
   appStore.fetchAppConfig()
+  createTransactionSigner()
```
Comment (ajithranka) [inline src/pages/AppConfigure.vue:111]: Why do we need to fetch the app config and re-initialised the transaction signer again here? It should already be initialised to the current app.

Comment (shrinathprabhu) [reply src/pages/AppConfigure.vue]: We need to fetch app config to reset the store with existing config. Currently all the changes made in the configure page changes the state of the app.store, so on cancel we need to reset it in order to be in sync with gateway values.

File: src/pages/AppConfigure.vue [hunk 2]
```diff
@@ -26,6 +26,7 @@ import {
   type ConfigureTab,
   type ConfigureTabType,
 } from '@/utils/constants'
+import { createTransactionSigner } from '@/utils/signerUtils'
```
Comment (ajithranka) [inline src/pages/AppConfigure.vue:29]: Remove unused import. Why wasn't this caught by our linter?

**Review by ajithranka — APPROVED** [review]

**Review by karthik-durai — APPROVED** [review]

File: src/pages/AppConfigure.vue (hunk 1 above)
Comment (ajithranka) [reply src/pages/AppConfigure.vue]: Okay, let's keep it until we move to using draft states for forms.

File: src/pages/AppConfigure.vue (hunk 2 above)
Comment (shrinathprabhu) [reply src/pages/AppConfigure.vue]: Resolved at https://github.com/arcana-network/developer-dashboard/pull/124/commits/2d52bc688d134b3021527fe121112a32d673d02e

**Pull request merged**